timesketch/api/v1/resources/sketch.py

```diff
@@ -64,9 +64,12 @@ def get(self):
         Returns:
             List of sketches (instance of flask.wrappers.Response)
         """
-        filtered_sketches = Sketch.all_with_acl().filter(
-            not_(Sketch.Status.status == 'deleted'),
-            Sketch.Status.parent).order_by(Sketch.updated_at.desc()).all()
+        if current_user.admin:
```

Comment:
Do we really want to list all sketches here? Wont this mess up the UI for all admin users?

Reply:
see screenshots sent to you offline.

API client has been changed to accommodate this, but I assume we need to make some UI changes, at least change the messaging, or have it in a similar way as the `archived` mode.. we can change the status to "admin_unknown" or something like that, and in the UI check if that's the status and then display a similar windows as you see when you visit an archived sketch... that is you can see the name/description of the sketch and then have the option of archiving it, that's it.. since ethat's all you're supposed to see anyway.

WDYT?

Can you make the UI change afterwards.. I can change the status or whatever needs to be done to make it easy to identify in the UI.

Reply:
ack, as discussed offline. sounds good.